ansible/playbooks/AdoptOpenJDK_Unix_Playbook/main.yml

```diff
@@ -36,21 +36,25 @@
     - ant                         # Testing
     - Ant-Contrib                 # Testing
     - maven                       # Testing
-    - GIT_Source
+    - role: GIT_Source
+      when: ansible_distribution != "Solaris" # Compile fails on Solaris
     - gmake
     - Docker                      # Testing
     - role: NVidia_Cuda_Toolkit   # OpenJ9
       tags: [build_tools, build_tools_openj9]
     - Superuser                   # AdoptOpenJDK Infrastructure
-    - Swap_File
+    - role: Swap_File
+      when: ansible_distribution != "Solaris" # These steps fail on Solaris
     - Crontab
     - NTP_TIME
     - gcc_48
     - role: gcc_7                 # OpenJ9
       tags: [build_tools, build_tools_openj9]
     - role: cmake                 # OpenJ9 / OpenJFX
+      when: ansible_distribution != "Solaris" # Compile fails on Solaris
       tags: [build_tools, build_tools_openj9, build_tools_openjfx]
-    - ccache
+    - role: ccache
+      when: ansible_distribution != "Solaris" # Not needed on Solaris
```

Comment:
We say "Not needed on Solaris" yet we still seem to have it in the common vars. Is this correct? I'd normally expect the ccache rol not to go through if it detects another version installed, so this guard seems irrelevant

Reply:
updated, the check only looks in `/usr/local/bin/ccache` which isn't where it's located on Solaris. I've updated the text.